org.eclipse.winery.topologymodeler.ui/src/app/node/node.component.ts

```diff
@@ -284,7 +280,6 @@ export class NodeComponent implements OnInit, AfterViewInit, OnDestroy, DoCheck
 
     /**
      * Checks if it was a click or a drag operation on the node.
-     * @param $event
      */
     mouseUpHandler($event): void {
```

Comment:
Please assign the `$event` its correspnding type. Applies for the whole file.

Reply:
Types are invoked automatically. Adding type annotations manually causes the typescript compiler to abort during bundling.